spec/classes/corosync_spec.rb

```diff
@@ -262,6 +242,7 @@
     let :facts do
       { osfamily:        'Debian',
```

Comment:
our newer rubocop config will fail here. could you move "osfamily:.." to the next line?